src/components/ResultCard/ResultCard.tsx

```diff
@@ -62,8 +62,10 @@ export default function ResultCard(props: ResultCardProps) {
 
   const resultCardContentWithLink = () => (
     <a
+      rel='noreferrer'
```

Comment:
Can we discuss with the team if we want noreferrer here?

Reply:
Sure, the thing is we shouldn't specify a `_blank` target attribute without setting the `rel` to either `noopener` or `noreferrer` (Security risk).

Reply:
We definitely need one of those. `noopener` might be the better choice here

Reply:
I don't think `noopener` is enough:

<img width="1024" alt="image" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/17954212/427e5daa-e7f8-4510-b1c0-4c51afb9d9a8">

@esezen @mocca102

Reply:
Interesting 🤔 Ok let's keep it as is. @HHHindawy can you resolve the conflicts? I can merge this today

Reply:
@esezen Done 👍